apps/api/src/domain/models/vocabulary-list/entities/vocabulary-list.entity.ts

```diff
@@ -47,14 +54,17 @@ export class VocabularyList extends Resource {
     })
     readonly entries: VocabularyListEntry[];
 
-    @NestedDataType(VocabularyListVariable, {
+    /**
+     * TODO rename this `property filters`
```

Comment:
[Here](https://www.pivotaltracker.com/story/show/186469235) is the story for that.